config/plotdefs-website/web-vitals-barchart.yaml

```diff
@@ -107,19 +107,19 @@ series:
 layout:
   barmode: "stack"
   title:
-    text: "Data: {{ .StartOfPreviousWeek | simpledate }} - {{ .EndOfPreviousWeek | simpledate }} ({{ .Params.region }})"
+    text: "Data: {{ .StartOfPreviousWeek | simpledate }} - {{ .EndOfPreviousWeek | simpledate }}<br>({{ .Params.type }}, {{ .Params.region }}, {{ .Params.website }})"
     font:
       size: 10
-    x: 0.01
```

Comment:
I actually prefer `x: 1`. This way the "Copy Link to Plot" and the title are right-aligned.